### DISCUSSION TITLE
Sequent '8-RELAYS for RASPBERRY PI' v5.3 board

### DISCUSSION THREAD

**Discussion by subcritical71** [tagyoureit/nodejs-poolController#693] [REM] [answered]
I just received the Sequent 8 v5.3 relay board. I can't seem to get it to connect in REM.

In the docs it has this note;
`Note1: Previous version of the 8-Relay card has I2C address range 0x38-0x3F`

The table for v5.3 shows using address range 0x20-0x27 ([8-RELAYS for RASPBERRY PI](https://cdn.shopify.com/s/files/1/0534/4392/0067/files/8-RELAYS-UsersGuide-V5.pdf?v=1677024406))

Should this board be working with the selection of v5 as is?

**Comment by rstrouse** [comment, marked as answer]
Those crazy kids at Sequent are at it again.  The Sequent 8 cards have never been able to auto detect due to the way they report.  Run i2cdetect to find the address and manually add it.  I assume they didn't re-arrange the relay bits (again).

**Reply by subcritical71** [reply]
Yeah, I figured. I'm still a bit lost. I have added the I2C Bus at #1 as the result of i2cdetect -l was;
`i2c-1	i2c       	bcm2835 (i2c@7e804000)          	I2C adapter
i2c-21	i2c       	Broadcom STB :                  	I2C adapter
i2c-20	i2c       	Broadcom STB :                  	I2C adapter`

iirc, 20 and 21 are for the rpi.

I then opened the 'I2C - Bus #1' tab and tried adding several addresses (3, 4, 27) and specifying the device as 'Multi-Relay Board'. (is active -> Save). Then I select the controller. I've tried every one. When I enable then activate a relay it does not respond. 

Still a little new to this but things are not going my way atm. Thanks for the help and fast response though. I really appreciate it.

**Reply by subcritical71** [reply]
Ok, looks like I got it. had to add the address at position 39 (0x27).

Thanks again.

**Reply by tagyoureit** [reply]
That's pretty interesting.  I have two relay cards and had trouble finding the address for the 2nd one initially, too.  I missed the documentation you pointed to earlier, but why they chose to go backwards from 0x27->0x20 for Board ID 1-8 is another Sequent mystery that may never be solved.

**Reply by rstrouse** [reply]
For future reference the i2cdetect command that you want is `i2cdetect -y 1`.  This will spit out all the addresses in hex.  Maybe one day I will allow you to enter new addresses in hex.
